test/suite/test_tiered19.py

```diff
@@ -116,6 +116,26 @@ def test_gcp_filesystem(self):
         self.assertRaisesHavingMessage(wiredtiger.WiredTigerError,
             lambda: ss.ss_customize_file_system(
                 session, no_access_bucket, None, self.get_fs_config(prefix)), err_msg)
+        
+         # We cannot use the file system to create files, it is readonly.
+        # So use python I/O to build up the file.
+        f = open('foobar', 'wb')
+
+        outbytes = ('MORE THAN ENOUGH DATA\n'*100000).encode()
+        f.write(outbytes)
+        f.close()
+
+        # Flushing copies the file into the file system.
+        self.assertEquals(ss.ss_flush(session, fs, 'foobar', 'foobar', None), 0)
+        self.assertEquals(ss.ss_flush_finish(session, fs, 'foobar', 'foobar', None), 0)
+
+        # Flush non valid file into GCP will result in an exception.
+        err_msg = "Exception: No such file or directory"
+        self.assertRaisesHavingMessage(wiredtiger.WiredTigerError,
+            lambda: ss.ss_flush(session, fs, 'non_existing_file', 'non_existing_file', None), err_msg)
+        # Check that file does not exist in GCP.
+        self.assertRaisesHavingMessage(wiredtiger.WiredTigerError,
+            lambda: ss.ss_flush_finish(session, fs, 'non_existing_file', 'non_existing_file', None), err_msg)
```

Comment:
We don't need to do it in this PR, but we should consider breaking this function into subfunctions (`check_customize_file_system`, `check_flush`) in the next ticket.